ecpp

```diff
@@ -112,7 +150,114 @@ def main():
     except KeyboardInterrupt:
         print(n)
         raise
-    print("Number is prime")
+    print("\n")
+    print("[+] number is prime\n")
+
+def correlate(moduli, certs=None, verify_opt=None, primo_opt=None):
+    """Correlate between functionalities"""
+    missing_cert_flag = False
+    cert_files = []
+    if certs is None:
+        certs = certs_dir
+    for path, _, files in os.walk(certs):
+        for name in files:
+            cert_files.append(os.path.join(path, name))
+    status = None
+    cert_dict = {}
+    for cert in cert_files:
+        if cert.endswith('.out'):
+            path = cert
+            cert_candidate = candidate_finder(path)
+            cert_dict[cert_candidate] = path
+    with open(str(moduli)) as f:
+        inputfile = filter(None, (line.rstrip() for line in f))
+        for count_line, line in enumerate(inputfile, 1):
+            if line.startswith('#'):
+                continue
+            prime_moduli = (line.split(" ")[6])
+            int_prime_moduli = int(prime_moduli, 16)
+            status = False
+            if int_prime_moduli in cert_dict:
+                path = cert_dict[int_prime_moduli]
+                print("[+] line {0}: {1}".format(count_line, path))
+                status = True
+                if verify_opt == True:
+                    verify(path)
+            if status == False:
+                print("[-] line {0}: no "
+                      "certificate found".format(count_line))
+                missing_cert_flag = True
+                if primo_opt:
+                    print("[+] line {0}: "
+                          "creating input file for "
+                          "{1}".format(count_line, primo_in))
+                    create_in_file(primo_in, prime_moduli)
+    return missing_cert_flag
+
+def usage():
+    """Usage"""
+    print("usage: eccp [-h] [-m arg | -i arg] [-c arg] [-p] [-v] \n")
+    print("positional arguments:\n")
+    print("-m, --moduli         specifie moduli file")
```

Comment:
typo: s/specifie/specify/, also noting that this is the SSH moduli file we're talking about would be a good idea, as well as this is the mode of operation in which the tool will verify that the numbers in moduli file are safe primes, not only that they are prime